tools/py_common/btree_format.py

```diff
@@ -86,7 +86,6 @@ class PageType(enum.Enum):
     '''
     WT_PAGE_INVALID = 0
     WT_PAGE_BLOCK_MANAGER = 1
-    WT_PAGE_COL_FIX = 2
```

Comment:
I'm not too familiar with how the files in the tools module are used, but since `WT_PAGE_COL_FIX` was changed to `WT_PAGE_COL_FIX_DEPRECATED` in `src/include/btmem.h`, should we also mark this enum value as deprecated rather than deleting it here?

If it’s safe to remove it entirely, then the current change is all good, I just wanted to double check for consistency with the rest of the FLCS deprecation work.